packages/design-system/src/components/web-components/ds-dropdown.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect } from 'react';
+import WebComponentDocTemplate from '../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { action } from '@storybook/addon-actions';
+import './ds-dropdown';
+
+const options = [
+  { label: 'Confederated Tribes and Bands of the Yakama Nation', value: '1' },
+  { label: 'Confederated Tribes of the Chehalis Reservation', value: '2' },
+  { label: 'Confederated Tribes of the Colville Reservation', value: '3' },
+];
+
+export default {
+  title: 'Web Components/Dropdown',
+  argTypes: {
+    autofocus: {
+      description: 'Sets the focus on the dropdown when it is first added to the document.',
+      control: 'boolean',
+    },
+    children: { control: false },
+    'class-name': {
```

Comment:
is this needed as a prop? users could add `class` to a root element instead of needing a special prop

Reply:
I'm just trying to keep it consistent with what we have so far. It might not be totally necessary, but it puts the classes on the same element as the `ds-c-dropdown`. We do the same with `ds-c-alert`. Maybe we can revist this when we pilot them.